components/list/list-item-checkbox-mixin.js

```diff
@@ -76,7 +77,13 @@ export const ListItemCheckboxMixin = superclass => class extends superclass {
 
 	_renderCheckboxAction(inner) {
 		return this.selectable ? html`
-			<div @click="${this._handleCheckboxActionClick}" class="d2l-checkbox-action" ?disabled="${this.disabled}">${inner}</div>
+			<label @click="${this._handleCheckboxActionClick}"
+				class="d2l-checkbox-action"
+				?disabled="${this.disabled}"
+				for="${this._checkboxId}"
+				tabindex="0">
```

Comment:
I agree this is a bit weird.

The way that the designs have defined the checkbox, because it lives in the `control` area rather than `control-action`, the input itself is not meant to receive focus - but your points make me want to confirm this with @m6jones. What do you think?

Reply:
Yup this is an issue. So focus should be brought to checkbox. I got so wrapped up in my action naming. 

I really wanted to treat all the areas the same. But that seems overly messy. I also rather the checkbox mixin not to worry about placing tabindex. 

So my thoughts right now is to change things up a bit and set focus based on what is in the control areas. Treat the content area the same as you are doing now. 

I would still prefer all the areas get treated the same.

Reply:
I've changed things to only prevent focus on the `content` area, as the dragger will also have focusability.